app/controllers/collections_controller.rb

```diff
@@ -16,7 +16,7 @@ def show
   def edit
     draft = CollectionDraft.create_from_collection(@collection, current_user, @native_id)
     Rails.logger.info("Audit Log: Draft for #{draft.entry_title} was created by #{current_user.urs_uid} in provider #{current_user.provider_id}")
-    flash[:success] = 'Draft was successfully created'
+    flash[:success] = 'Collection Draft Created Successfully!'
```

Comment:
It looks like you added all of these translations to en.yml but you're not utilizing it here.